src/main/java/seedu/socket/logic/commands/ClearCommand.java

```diff
@@ -15,8 +15,8 @@ public class ClearCommand extends Command {
 
     public static final String COMMAND_WORD = "clear";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Clears all persons or persons whose tag "
-            + "contain any of the keyword(s) (case-insensitive) in the tag fields.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Clears all persons or persons whose tag(s) "
```

Comment:
There is a typo "persons or persons".

Reply:
Its persons, or persons whose tag, but i'll add a comma for separation, thanks